llvm/lib/IR/RepoTicket.cpp

```diff
@@ -118,25 +127,46 @@ static GODigestState accumulateGOFinalDigestOrInitialDigestAndGetGODigestState(
 
 // Update the GO's hash value by adding the hash of its dependents.
 template <typename FunctionType>
-static bool updateDigestUseDependenciesAndContributions(
-    const GlobalObject *GO, MD5 &GOHash, GOStateMap &Visited, GOInfoMap &GOIMap,
-    FunctionType AccumulateGODigest) {
+static auto updateDigestUseDependenciesAndContributions(
+    const GlobalObject *GO, GOStateMap &Visited, GOInfoMap &GOIMap,
+    MemoizedHashes &GOHashCache, FunctionType AccumulateGODigest)
+    -> std::tuple<bool, size_t, DigestType> {
```

Comment:
Please can you comment what the elements of the tuple represent?

Reply:
Added the comment.